jenkins/env_setup/Jenkinsnodedetails.groovy

```diff
@@ -0,0 +1,41 @@
+def count = 0
+try {
+    for (Node n : Jenkins.get().getNodes()) {
+        try {
+            if (n.getComputer().isOffline() == true) {
+                count ++
+                hudson.plugins.jobConfigHistory.ConfigInfo config = n.getComputer().actions[0].getSlaveConfigs()[0]
+                println('Sr.no:' + count + ' Node_name:' + n.name + ' Host:' + n.launcher.host + ' Label:' + n.getLabelString() + ' User:' + config.getUser() + ' UserId:' + config.getUserID())
```

Comment:
Instead of adding `Sr.no`, `Node_name:` in each line add it in first-line and then only print values. Report should look like this 
```
Sr.no,Node_name,Host,Label,User,UserId
146,ssc-vm-g4-rhev4-0189.colo.seagate.com,ssc-vm-g4-rhev4-0189.colo.seagate.com,ssc-vm-g4-rhev4-0189.colo.seagate.com,Ketan,Arlulkar,528894
147,ssc-vm-g4-rhev4-0258,10.230.240.158,ssc-vm-g4-rhev4-0258,Divya,Kachhwaha,Kachhwaha,532698
148,ssc-vm-g4-rhev4-0718.colo.seagate.com,ssc-vm-g4-rhev4-0718.colo.seagate.com,ssc-vm-g4-rhev4-0718,Gaurav,Chaudhari,934370
149,ssc-vm-g4-rhev4-0896.colo.seagate.com,ssc-vm-g4-rhev4-0896.colo.seagate.com,ssc-vm-g4-rhev4-0896.colo.seagate.com,Divya,Kachhwaha,Kachhwaha,532698
```

Reply:
I have done changes as per suggestions.